docs/src/main/asciidoc/_templates/template-concepts.adoc

```diff
@@ -6,10 +6,10 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 TODO: See Quarkus documentation reference for details
 - Title should have an implied "Understanding..." in front. 
-- ID should start with 'concept-'
+- ID should end with '-concepts' (using the filename works)
 - choose appropriate categories
 ////
-[id="concept-..."]
+[id="...-concepts"]
```

Comment:
I noticed that 'concepts' is plural but the other template types are singular? 

To avoid confusion and inconsistencies, would it make sense to change this?

Reply:
will have another pass, yes